io.cucumber.eclipse.targetdefinition/cucumber.eclipse.targetdefinition.target

```diff
@@ -100,6 +100,24 @@ Export-Package:        *;version="${version}";-noimport:=true
 			</dependency>
 		</dependencies>
 	</location>
+	<location includeDependencyDepth="infinite" includeDependencyScopes="compile" includeSource="true" missingManifest="generate" type="Maven">
+		<dependencies>
+			<dependency>
+				<groupId>io.cucumber</groupId>
+				<artifactId>gherkin-utils</artifactId>
+				<version>9.0.0</version>
+				<type>jar</type>
+			</dependency>
+		</dependencies>
+		<instructions><![CDATA[
+Bundle-Name:           ${mvnGroupId}:${mvnArtifactId}:${mvnVersion}
+version:               ${version_cleanup;${mvnVersion}}
+Bundle-SymbolicName:   ${mvnGroupId}.${mvnArtifactId}
+Bundle-Version:        ${version}
+Import-Package:        !sun.*,io.cucumber.messages.internal.com.google.gson*,!io.cucumber.messages.internal.*,!org.checkerframework.*,*
```

Comment:
I don't think `cucumber.messages.internal` still exists.

Reply:
@mpkorstanje year this is maybe not required anymore and is a bit cumbersome to manage anyways. Because of that I would suggest adding the data in cucumber directly as with:

- https://github.com/cucumber/gherkin-utils/pull/131

this is much more appropriate and consistent.